packages/coinstac-ui/app/main/index.js

```diff
@@ -18,6 +18,14 @@ const ipcPromise = require('ipc-promise');
 const mock = require('../../test/e2e/mocks');
 const ipcFunctions = require('./utils/ipc-functions');
 
+// Require winston logger to log docker issues 
+const winston = require('winston');
+
+const winstonLogger = new winston.Logger({
+  level: 'verbose',
+  transports: [new winston.transports.Console()],
+});
+
```

Comment:
Good setting up Winston, but there is already a logger instance setup that you should use. Look around line 75